src/requirements/data/majors/envE.ts

```diff
@@ -0,0 +1,147 @@
+import { CollegeOrMajorRequirement, Course } from '../../types';
+import { includesWithSingleRequirement, includesWithSubRequirements } from '../checkers-common';
+
+const majorApproved: readonly string[] = [
+  'BEE',
+  'CEE',
+  'CHEME',
+  'EAS',
+  'MAE',
+  'PLSCS',
+  'ENMGT',
+  'ECON',
+  'AEM',
+  'ENGRG',
+  'STS',
+];
+
+const envEngineeringRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Organic Chemistry',
+    description: 'CHEM 1570 (recommended) or 3530, 3570',
+    source:
+      'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19794#environmentalengineering',
+    checker: includesWithSubRequirements(['CHEM 1570', 'CHEM 3530', 'CHEM 3570']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+  },
+  {
+    name: 'Biological Sciences',
+    description:
+      'Choose one of the following Introductory Biology courses: BIOEE/BIOSM 1610, BIOMG 1350, BIOEE/BIOSM 1780, BIOG 1440 or 1445',
+    source:
+      'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19794#environmentalengineering',
+    checker: includesWithSingleRequirement(
+      'BIOEE 1610',
+      'BIOMG 1350',
+      'BIOEE 1780',
+      'BIOG 1440',
+      'BIOG 1445'
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+  },
+  {
+    name: 'Engineering Distribution',
+    description:
+      'ENGRD 2510. ENGRD 2020 is recommended for the second Engineering Distribution course',
+    source:
+      'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19794#environmentalengineering',
+    checker: includesWithSubRequirements(['ENGRD 2510']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    allowCourseDoubleCounting: true,
+  },
```

Comment:
Are we purposefully not including ENGRD 2020 for the second slot?

Reply:
yea its only college reqs (i just added this as a reminder). maybe eventaully we can move engrd to just majors